### PR TITLE
refactor v1

### PR DESCRIPTION
**Pull request by ciehanski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea
 cmd/globalcv/env.sh
 cmd/globalcv/globalcv
+routes.md
```
Comment (kuskoman) [inline .gitignore]: why are you ignoring routes.md?

File: cmd/globalcv/env.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```
Comment (kuskoman) [inline cmd/globalcv/env.sh:1]: I'd rather it to be .env file read from go code, so i can use it on windows
eventually we can create powershell file along with bash file

**Review by kuskoman — DISMISSED** [review]

File: cmd/globalcv/env.sh (hunk 2 above)
Comment (ciehanski) [reply cmd/globalcv/env.sh:1]: works for me

File: .gitignore (hunk 1 above)
Comment (ciehanski) [reply .gitignore]: temporarily since I just added the contents to README

**Pull request closed without merging**